src/github.com/couchbase/sync_gateway/db/channel_cache.go

```diff
@@ -186,55 +224,67 @@ func (c *channelCache) _appendChange(change *LogEntry) {
 	end := len(log) - 1
 	if end >= 0 {
 		if change.Sequence <= log[end].Sequence {
-			base.LogTo("Cache", "LogEntries.appendChange: out-of-order sequence #%d (last is #%d) - handling as insert",
-				change.Sequence, log[end].Sequence)
+			//base.LogTo("Cache+", "LogEntries.appendChange: out-of-order sequence #%d (last is #%d) - handling as insert",
```

Comment:
this commented log is a bit confusing (was it commented because it's no longer true?).  I think we should enable it or remove it.